app/models/user.rb

```diff
@@ -1,5 +1,8 @@
 # An application user, uses the `devise` library
 class User < ApplicationRecord
+  # Every user can have multiple social accounts like GitHub, Telegram, ...
+  has_many :social_accounts
```

Comment:
Possible edge case: Do we have any check to make sure that you cannot assign two Twitter handles to one account? Or do we do not want such a limitation at all?

Reply:
Our PO liked the absence of this limitation.